app/components/conceptAdd/conceptAdd.controller.js

```diff
@@ -14,9 +14,11 @@
 		.module('conceptDictionaryApp')
 		.controller('ConceptAddController', ConceptAddController)
 		
-	ConceptAddController.$inject = ['serverLocales', 'loadClasses', 'loadDataTypes', 'conceptsService'];
+	ConceptAddController.$inject = 
+		['serverLocales', 'loadClasses', 'loadDataTypes', 'conceptsService', '$location', '$routeParams'];
 		
-	function ConceptAddController(serverLocales, loadClasses, loadDataTypes, conceptsService){
+	function ConceptAddController
+			(serverLocales, loadClasses, loadDataTypes, conceptsService, $location, $routeParams){
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This function has too many statements. (41)](https://www.codacy.com/app/openmrs/openmrs-owa-conceptdictionary/file/2347631265/issues/source?bid=3196969&fileBranchId=3209439#l21)